pkg/engine/const.go

```diff
@@ -38,8 +38,6 @@ const (
 	NetworkPluginKubenet = "kubenet"
 	// NetworkPluginFlannel is the string expression for flannel network plugin
 	NetworkPluginFlannel = "flannel"
-	// HeapsterAddonName is the name of the kube-heapster-deployment addon
-	HeapsterAddonName = "kube-heapster-deployment"
```

Comment:
why is this not needed anymore but not the other addon names in this file?

Reply:
It's a bug, there is no heapster addon with that name. Basically, the current "get user-configurable `Data`" foo doesn't work for heapster.